fix: ldap allow case sensitivity settings to force dn and memberOf to… (#26923)
backstage/backstage: docs/integrations/ldap/org.md

  members: member
```

## Optional Vendor Configuration

> In case the LDAP vendor isn't available as a provided configuration, A new optional `vendor` configuration section is available which allows overriding the location for `dn` and `uuid` settings, and a case sensitivity function.

Reviewer: ```suggestion
In case the LDAP vendor isn't automatically detected by the module, an optional `vendor` configuration section is available which allows overriding the location for `dn` and `uuid` settings, and a case sensitivity setting.
```

#### vendor.dnAttributeName

> Allows ability to override non standard `entryDN` values.

Reviewer: ```suggestion
Allows explicitly defining the name of the attribute that stores each entry's DN.
```

#### vendor.uuidAttributeName

> Allows ability to override non standard `entryUUID` values.

Reviewer: ```suggestion
Allows explicitly defining the name of the attribute that stores each entry's UUID.
```

#### vendor.dnCaseSensitive

> Provides the ability to ignore case sensitivity issues with user/group mappings. If set to true will still link user/members to groups by forcing the inbound values from `dn`, `member`, `memberOf` to lowercase.

Reviewer: ```suggestion
Provides the ability to ignore case sensitivity issues with user/group mappings. If set to true, the ingestion will link user/members to groups whether their `dn`, `member`, or `memberOf` values have the right case or not.
```
